js/start.js

```diff
@@ -622,6 +631,9 @@ function start() {
               }
             });
           }
+
+          // add the feedback mechanism
+          addFeedback();
```

Comment:
The feedback was moved because previously it was created before the profile and settings were available, leaving all of that data unreported.

Reply:
Definitely not holding up merge for this... just musing...

Now that it's moved, doesn't that mean a user can't give feedback during the startup portion of the app (which could be lengthy)? I would have thought it would be fine to allow feedback earlier and if we have the profile and settings, they are sent over, otherwise not. I'm assuming that if the user didn't submit feedback until after they became available, then they would be sent?